frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -168,6 +169,17 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
         dispatch(machinery.actions.get(source, locale, pk));
     }
 
+    copyLinkToClipboard = () => {
+        const { locale, project, resource, entity } = this.props.parameters;
+
+        let string_link = `https://pontoon.mozilla.org/${locale}/${project}/${resource}/?string=${entity}`
```

Comment:
That's better. But it will only work on Mozilla's instance of Pontoon, which runs on `pontoon.mozilla.org`. There are other organizations that use self-hosted installs of Pontoon.